FileAPI/blob/Blob-slice.html

```diff
@@ -35,7 +35,24 @@
     type: "content/type",
     desc: "blob2."
   });
+  
+  test_blob(function() {
+    return blob3 = new Blob().slice(0,0,null);
```

Comment:
Apparently the reason that blob1 and blob2 are assigned above is that they're used in later tests. Since these aren't, you can just `return new Blob()...`